module/Calendar.js

```diff
@@ -0,0 +1,311 @@
+class Calendar {
```

Comment:
For better readability, please use a tab consisting of two spaces

Reply:
Also use _eslint_ to check the code

Reply:
OK, now i'm using two spaces.